### DISCUSSION TITLE
Cannot pass cloudflare with checkbox

### DISCUSSION THREAD

**Discussion by NCLnclNCL** [ultrafunkamsterdam/undetected-chromedriver#1205] [General]
I test in https://xamvn.gay with checkbox, but I have a loop and can't get through it

**Comment by ultrafunkamsterdam** [comment]
I get this on my phone in regular browser as well
![Screenshot_20230418-165931](https://user-images.githubusercontent.com/21027969/232818876-384ce63b-855c-484b-abd5-2c105acce734.jpg)

**Reply by NCLnclNCL** [reply]
But i use selenium, when i click it, it will be looped like that forever
